doc/api/modules.md

```diff
@@ -160,7 +160,8 @@ require(X) from module at path Y
    a. LOAD_AS_FILE(Y + X)
    b. LOAD_AS_DIRECTORY(Y + X)
 4. LOAD_NODE_MODULES(X, dirname(Y))
-5. THROW "not found"
```

Comment:
I think we should switch these around now?

Reply:
They have been, exactly.